### PR TITLE
Update the list of cluster configuration resources.

### PR DESCRIPTION
**Pull request by sanchezl**

### CODE REVIEW

File: architecture/customizations.adoc [hunk 1]
```diff
@@ -25,27 +25,49 @@ Custom Resource Definitions (CRDs) that are deployed on your cluster.
 |===
 |Configuration resource |Description
 
+|APIServer
+|Configure the API server.
```
Comment (deads2k) [inline architecture/customizations.adoc]: Kubernetes API server I'd think.

File: architecture/customizations.adoc [hunk 2]
```diff
@@ -25,27 +25,49 @@ Custom Resource Definitions (CRDs) that are deployed on your cluster.
...
 |Network
-|See
+|Configure the cluster networks.
 
 |OAuth
-|See
+|Configure the integrated OAuth server.
+
+|Project
+|Configure how projects are created.
```
Comment (deads2k) [inline architecture/customizations.adoc]: @sanchezl this one is also ours to doc, you should make sure the jira doc issue has subtasks for it

Comment (sanchezl) [reply architecture/customizations.adoc]: Opened https://jira.coreos.com/browse/MSTR-356

File: architecture/customizations.adoc (hunk 1 above)
Comment (sanchezl) [reply architecture/customizations.adoc]: Updated

**Comment by deads2k** [comment]
@kalexand-rh if we're going to keep this page, you're trying to stay up to date with this list here: https://github.com/openshift/api/blob/master/config/v1/register.go#L32-L64 

@sanchezl you're missing a couple.  Skip ClusterOperator and ClusterVersion

**Comment by deads2k** [comment]
Once that's update, perhaps merge this and expand on the APIServer in a different pull.

**Comment by sanchezl** [comment]
@deads2k I was missing Scheduler, this list now matches.

**Comment by kalexand-rh** [comment]
@deads2k, this list is supposed to match what's showing up in the *Administration* > *Cluster Settings* > *Global Configuration* page. Neither the PR nor the current docs match today's build. Do you know when this list will be stable?

**Pull request merged**